ros2cli/ros2cli/helpers.py

```diff
@@ -107,3 +108,13 @@ def unsigned_int(string):
     if value < 0:
         raise ArgumentTypeError('value must be non-negative integer')
     return value
+
+
+def collect_stdin():
+    lines = b""
```

Comment:
I don't know why our linters didn't pick this up, but the larger CI is showing a problem here.  This should be:

```suggestion
    lines = b''
```

Reply:
Fixed.